client/components/Editor/styles/table.scss

```diff
@@ -14,20 +24,17 @@ td {
 		margin-bottom: 0;
 	}
 }
+
 /* This clunky table, tableWrapper thing is because */
```

Comment:
This comment (or at least its specific wording) appears to be out of date following these changes.

Reply:
Got it, I'll update it or remove it.